remove (unused) swap16/swap32 causing a build failure

merged upstream for the next release

Index: engines/saga2/cmisc.h
--- engines/saga2/cmisc.h.orig
+++ engines/saga2/cmisc.h
@@ -34,19 +34,6 @@ int32 clamp(int32 lowerLimit, int32 num, int32 upperLi
 //  Converts x/y point to angle + distance
 int16 ptToAngle(int16 dx, int16 dy, int16 *dist = NULL);
 
-//  Byte-swap a 16-bit integer
-inline int16 swap16(uint16 w) {
-	return (w >>  8) | (w <<  8);
-}
-
-//  Byte-swap a 32-bit integer
-inline int32 swap32(int32 l) {
-	return ((l >> 24) & 0x000000ffL) |
-	       ((l >>  8) & 0x0000ff00L) |
-	       ((l <<  8) & 0x00ff0000L) |
-	       ((l << 24) & 0xff000000L);
-}
-
 #if     defined( USEWINDOWS )
 inline void delay(int32 ms)        {
 	Sleep(ms);
